src/main/java/frc/robot/subsystems/ShooterSubsystem.java

```diff
@@ -85,6 +85,8 @@ public ShooterSubsystem() {
         setPIDGains(kP1.get(), kI1.get(), kD1.get(), 1);
         setFFGains(kFF1.get(), 1);
 
+        m_motor.burnFlash();
```

Comment:
Can we have the similar method of burnFlash() as other subsystems here? (i.e. after some time, then burnFlash)?
Instead of directly call motor's burnFlash?